.nvmrc

```diff
@@ -0,0 +1 @@
+20
```

Comment:
We should start using 22 as the new default as 20 is in maintenance already.

Reply:
Bumped to 22 in [c7a1ca5](https://github.com/hemilabs/tge-mint-page/pull/1/commits/c7a1ca508820baea55ce1c8535c5bbdb4a3c61d8)